src/cmd/run.js

```diff
@@ -256,6 +273,19 @@ export default async function run(
         );
       }
 
+      readline.emitKeypressEvents(process.stdin);
```

Comment:
This would be probably better moved inside the `if (process.stdin.isTTY) { ... }` below.